ingest/scripts/prepare_metadata.py

```diff
@@ -36,18 +36,18 @@ class Config:
 
 def split_authors(authors: str) -> str:
```

Comment:
```suggestion
def reformat_authors_from_genbank_to_loculus(authors: str) -> str:
```

The function comment probably needs to be rewritten as the operation is now much simpler: effectively we just need to replace every second comma with a semicolon `;`